### PR TITLE
CORE-9318 Separate DB users for automatic bootstrap and runtime

### PR DESCRIPTION
**Pull request by jacob-scott3141**
Updated changes to work with REST / RPC  name changes

### CODE REVIEW

File: .ci/e2eTests/Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@Library('corda-shared-build-pipeline-steps@5.0.1') _
+@Library('corda-shared-build-pipeline-steps@jacob/CORE-9318') _
```
Comment (yarivus) [inline .ci/e2eTests/Jenkinsfile]: please ensure this is changed before the merge ;)

Comment (jacob-scott3141) [inline .ci/e2eTests/Jenkinsfile]: Need to revert this before merge, after pipeline changes

File: values.yaml [hunk 2]
```diff
@@ -26,6 +26,15 @@ bootstrap:
   initialAdminUser:
     password:
       value: "admin"
+  db:
+    cluster:
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: "prereqs-postgresql"
+            key: "password"
```
Comment (davidcurrie) [inline values.yaml]:
```suggestion
            key: "postgres-password"
```

File: values.yaml [hunk 3]
```diff
@@ -26,6 +26,15 @@ bootstrap:
...
+      username:
+        value: "user"
```
Comment (davidcurrie) [inline values.yaml]: No reason why this should be different from what we have in the CI.
```suggestion
        value: "postgres"
```

File: values-prereqs.yaml [hunk 4]
```diff
@@ -18,6 +18,15 @@ bootstrap:
...
       value: "admin"
+  db:
+    cluster:
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: "prereqs-postgresql"
+            key: "password"
+      username:
+        value: "user"
```
Comment (davidcurrie) [inline values-prereqs.yaml]: Just to keep people on their toes, I named these differently in the new `corda-dev-prereqs` chart. You should check that these overrides work there.
```suggestion
            name: "prereqs-postgres"
            key: "postgres-password"
      username:
        value: "postgres"
```

**Review by davidcurrie — CHANGES_REQUESTED** [review]

**Review by yarivus — COMMENTED** [review]
see pending comment

**Review by davidcurrie — DISMISSED** [review]
👍

**Review by davidcurrie — APPROVED** [review]
👍

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**